var/spack/repos/builtin/packages/py-python-fmask/package.py

```diff
@@ -29,3 +30,5 @@ class PyPythonFmask(PythonPackage):
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
     depends_on("gdal+python", type=("build", "run"))
+
+    maintainers("gillins", "neilflood")
```

Comment:
For "readability", please move this higher, preferably before the license directive, so it doesn't get lost under the dependencies.